packages/@aws-cdk/aws-synthetics/README.md

```diff
@@ -1,16 +1,206 @@
-## AWS::Synthetics Construct Library
+## Amazon CloudWatch Synthetics Construct Library
+
 <!--BEGIN STABILITY BANNER-->
 ---
 
 ![cfn-resources: Stable](https://img.shields.io/badge/cfn--resources-stable-success.svg?style=for-the-badge)
 
 > All classes with the `Cfn` prefix in this module ([CFN Resources](https://docs.aws.amazon.com/cdk/latest/guide/constructs.html#constructs_lib)) are always stable and safe to use.
 
+![cdk-constructs: Experimental](https://img.shields.io/badge/cdk--constructs-experimental-important.svg?style=for-the-badge)
+
+> The APIs of higher level constructs in this module are experimental and under active development. They are subject to non-backward compatible changes or removal in any future version. These are not subject to the [Semantic Versioning](https://semver.org/) model and breaking changes will be announced in the release notes. This means that while you may use them, you may need to update your source code when upgrading to a newer version of this package.
+
 ---
 <!--END STABILITY BANNER-->
 
-This module is part of the [AWS Cloud Development Kit](https://github.com/aws/aws-cdk) project.
+Amazon CloudWatch Synthetics allow you to monitor your application by generating **synthetic** traffic. The traffic is produced by a **canary**. A canary is a configurable script that runs on a schedule. You configure the canary script to follow the same routes and perform the same actions as a customer, which allows you to continually verify your customer experience even when you don't have any customer traffic on your applications.
+
+## Canary
+
+To illustrate how to use a canary, assume your application defines the following endpoint:
+
+```bash
+curl https://api.awsomesite.com/user/books/topbook/
+The Hitchhikers Guide to the Galaxy
+
+```
+
+The below code defines a canary that will hit the `books/topbook` endpoint every 5 minutes:
+
+> **Not Implemented:** the `test` property will be a part of Milestone 2.
+
+```ts
+const canary = new Canary(this, 'MyCanary', {
+  name: 'mycanary'
+  frequency: Duration.minutes(5),
+  test: synth.Test.custom({
```

Comment:
Unsure about the `synth` name here. People could mistake it for the `cdk synth` meaning? It's also not introduced, so it could be anything.

Reply:
I hadn't thought of that. I'll introduce the import as `import * as synthetics from '@aws-cdk/aws-synthetics';` and use `synthetics` instead of `synth` to avoid confusion.